### PR TITLE
Add -DskipTests to dev docs

### PR DESCRIPTION
**Pull request by ash211**

### CODE REVIEW

**Review by foxish — APPROVED** [review]

File: resource-managers/kubernetes/README.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ important matters to keep in mind when developing this feature.
...
 
-    build/mvn compile -Pkubernetes -pl resource-managers/kubernetes/core -am
+    build/mvn compile -Pkubernetes -pl resource-managers/kubernetes/core -am -DskipTests
 
 To build a distribution of Spark with Kubernetes support, use the `dev/make-distribution.sh` script, and add the
 `kubernetes` profile as part of the build arguments. Any other build arguments can be specified as one would expect when
 building Spark normally. For example, to build Spark against Hadoop 2.7 and Kubernetes:
 
-    dev/make-distribution.sh --tgz -Phadoop-2.7 -Pkubernetes
+    dev/make-distribution.sh --tgz -Phadoop-2.7 -Pkubernetes -DskipTests
```
Comment (mccheah) [inline resource-managers/kubernetes/README.md]: I think, but am not positive, that make-distribution skips tests by default. Let's check the script to see.

Comment (mccheah) [reply resource-managers/kubernetes/README.md]: Yep just verified that - no need for skipTests here

**Comment by ash211** [comment]
@mccheah updated -- ready to merge?

**Comment by mccheah** [comment]
Looks fine, merge when the build is green.

**Comment by foxish** [comment]
Looking good, merging.

**Pull request merged**